core/lib/generators/solidus/install/templates/config/initializers/spree.rb.tt

```diff
@@ -42,6 +42,11 @@ Spree.config do |config|
   # their previous location first.
   config.redirect_back_on_unauthorized = true
 
+  # Set this configuration to `false` to allow promotions
+  # with no associated actions to be active for use by customers.
+  # See https://github.com/solidusio/solidus/issues/2777 for more info.
```

Comment:
Can you link to the new PR here instead?

Reply:
👍